initScripts/aarch32/Ubuntu_16.04/Docker_18.03.sh

```diff
@@ -137,19 +136,23 @@ initialize_swap() {
 docker_install() {
   echo "Installing docker"
 
+  # installing arm64 docker, since the machine is of arm64 architecture
   add-apt-repository -y "deb [arch=arm64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" && apt-get update
 
   install_docker="apt-get install -q --force-yes -y -o Dpkg::Options::='--force-confnew' docker-ce=$DOCKER_VERSION~ce-0~ubuntu"
   exec_cmd "$install_docker"
 
-  get_static_docker_binary="wget https://download.docker.com/linux/static/stable/aarch64/docker-$DOCKER_VERSION-ce.tgz -P /tmp/docker"
+  get_static_docker_binary="wget https://download.docker.com/linux/static/stable/armhf/docker-$DOCKER_VERSION-ce.tgz -P /tmp/docker"
```

Comment:
I think we should add some comments on exactly what we are achieving here. The flow is unusual.